sonic_platform_base/psu_base.py

```diff
@@ -223,6 +223,26 @@ def get_maximum_supplied_power(self):
         """
         raise NotImplementedError
 
+    def get_psu_power_warning_threshold(self):
```

Comment:
this naming is confusing as no warning is raised for this threshold. can we name it as "warning-suppress" level?

Reply:
Changed to `get_psu_power_warning_suppress_threshold`

Reply:
@stephenxs pr description still has warning threshold.

Reply:
@prgeor  you were probably checking an out-of-data commit. Use link https://github.com/sonic-net/sonic-platform-common/pull/304/files to open the latest one